src/main/java/uk/gov/hmcts/darts/arm/service/impl/UnstructuredToArmProcessorImpl.java

```diff
@@ -55,7 +55,7 @@ public UnstructuredToArmProcessorImpl(ExternalObjectDirectoryRepository external
                                           FileOperationService fileOperationService,
                                           ArchiveRecordService archiveRecordService) {
         super(objectRecordStatusRepository, userIdentity, externalObjectDirectoryRepository, externalLocationTypeRepository, dataManagementApi,
-              armDataManagementApi, fileOperationService);
+              armDataManagementApi, fileOperationService, 0);
```

Comment:
We should not hardcode it in the code even if its single mode. Have it passed in the constructor itself.